x-pack/legacy/plugins/siem/public/components/ml_popover/jobs_table/jobs_table.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import chrome from 'ui/chrome';
+import React, { useEffect, useState } from 'react';
+
+import {
+  CENTER_ALIGNMENT,
+  EuiBasicTable,
+  EuiButton,
+  EuiEmptyPrompt,
+  EuiLink,
+  EuiText,
+} from '@elastic/eui';
+
+import styled from 'styled-components';
+import * as i18n from '../translations';
+import { JobSwitch } from './job_switch';
+import { Job } from '../types';
+
+const JobNameWrapper = styled.div`
+  margin: 5px 0;
+`;
+
+const getJobsTableColumns = (
+  isLoading: boolean,
+  onJobStateChange: (jobName: string, latestTimestampMs: number, enable: boolean) => void
+) => [
+  {
+    name: i18n.COLUMN_JOB_NAME,
+    render: ({ id, description }: Job) => (
+      <JobNameWrapper>
+        <EuiLink href={`${chrome.getBasePath()}/app/ml`} target="_blank">
+          <EuiText size="s">{id}</EuiText>
+        </EuiLink>
+        <EuiText color="subdued" size="xs">
+          {description}
+        </EuiText>
+      </JobNameWrapper>
+    ),
+  },
+
+  {
+    name: i18n.COLUMN_RUN_JOB,
+    render: (job: Job) => (
+      <JobSwitch job={job} isSummaryLoading={isLoading} onJobStateChange={onJobStateChange} />
```

Comment:
Discussed this further with Michael -- current behavior is auto-install (on component mount, so any SIEM page load) of any missing jobs that are compatible with the current user's configured indices and so long as they have the necessary permissions as well. 

Let's discuss further though about the ability to show the uninstalled jobs and only install on `enable` as it may be confusing to users why these ML jobs keep showing up after they delete them. 😅